CMakeLists.txt

```diff
@@ -2,7 +2,11 @@ cmake_minimum_required(VERSION 3.2)
 
 set(MACOSX_BUNDLE_GUI_IDENTIFIER "org.navitproject.navit")
 set(MACOSX_BUNDLE_BUNDLE_NAME "Navit")
+set(MACOSX_BUNDLE_ICON_FILE "Default.png")
+execute_process(COMMAND git describe --tags OUTPUT_VARIABLE MACOSX_BUNDLE_INFO_STRING)
```

Comment:
- It should be handled if git is not available and if the build isn't run from a git repo (tar source package for example)
- Variable should be renamed to GIT_VERSION so if can also be used within config.h.in

Reply:
First of all thank you for the feedback. I'm working on it.